specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -1129,32 +1132,47 @@
         }
       }
     },
-    "ProxyResource": {
+    "PrivateLinkProxyResource": {
       "properties": {
         "id": {
           "readOnly": true,
           "type": "string",
-          "description": "Resource ID."
+          "description": "Azure resource Id"
         },
         "name": {
           "readOnly": true,
           "type": "string",
-          "description": "Resource name."
+          "description": "Azure resource name"
         },
         "type": {
           "readOnly": true,
           "type": "string",
-          "description": "Resource type."
+          "description": "Azure resource type"
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Resource tags"
+          "x-ms-azure-resource": true,
+          "description": "azure resource tags"
         }
       },
-      "description": "Common properties of proxy resource."
+      "description": "An azure resource object"
+    },
+    "ErrorResponse": {
+      "description": "Describes the format of Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
```

Comment:
you forgot details which is an array of ErrorResponse.
can you please copy whatever's in the common

Reply:
this definition I put here is the same as ["ErrorResponse" in existing spec](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/autoscale_API.json#L793)

there will be conflict when I try to generate dotnet SDK if the definition is non-identical

Reply:
Okay, but we use the details property to expose more details on the error.

Reply:
I can add one more definition of something like "ErrorResponseExtension" to be inherited from "ErrorResponse" and have "details"/"additionalInfo", what do you think?

Reply:
correct me if I was wrong, the ErrorResponse would be referenced only in ErrorResponseException, does it matter if the ErrorResponse doesn't contain the "detial"/"additionalInfo"?

Reply:
This is how the error response object should look:

public class ErrorResponse
    {
        [JsonProperty("code")]
        public string Code { get; set; }

        [JsonProperty("message")]
        public string Message { get; set; }

        [JsonProperty("target")]
        public string Target { get; set; }

        [JsonProperty("details"]
        public ErrorResponse[] Details { get; set; }
    }

If we don't contain "details" then user will get a partiall error response.

Reply:
I add ErrorResponseCommon below to have the "details" field, and the definition should be identical with the one defined in common-types.

BTW I don't have code generation environment setup in this laptop, will verify it manually with this new added block tomorrow once I got to the office.

Reply:
ok